src/main/java/seedu/address/logic/commands/FindPropertyCommand.java

```diff
@@ -4,31 +4,36 @@
 
 import seedu.address.commons.core.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.property.PropertyContainsKeywordsPredicate;
+import seedu.address.model.property.PropertyPredicateList;
 
 /**
- * Finds and lists all properties in address book containing any of the argument keywords.
+ * Finds and lists all properties in address book containing any of the argument keywords or by price
+ * given.
  * Keyword matching is case insensitive.
  */
 public class FindPropertyCommand extends Command {
 
     public static final String COMMAND_WORD = "find property";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all properties containing any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " hdb jurong";
-
-    private final PropertyContainsKeywordsPredicate predicate;
-
-    public FindPropertyCommand(PropertyContainsKeywordsPredicate predicate) {
-        this.predicate = predicate;
+            + "the specified keywords (case-insensitive) and/or by giving a price and search for properties "
+            + "with asking prices above or below that amount, then displaying them as a list with index "
+            + "numbers.\n"
+            + "Parameters: KEYWORD [MORE_KEYWORDS]... [pl/upper price limit(inclusive)] "
+            + "[pm/lower price limit(inclusive)\n"
```

Comment:
Perhaps this can be changed to:
`"Parameters: [KEYWORD]... [pl/UPPER_PRICE_LIMIT] [pm/LOWER_PRICE_LIMIT]"`
instead and put the note on inclusive in the description? So that it aligns with our command format of replacing the uppercase characters in the command format with user input.